dm/pkg/schema/tracker.go

```diff
@@ -221,6 +221,21 @@ func (tr *Tracker) AllSchemas() []*model.DBInfo {
 	return filteredSchemas
 }
 
+// ListSchemaTables lists all tables in the schema.
+func (tr *Tracker) ListSchemaTables(schema string) ([]string, error) {
+	allSchemas := tr.AllSchemas()
+	for _, db := range allSchemas {
+		if db.Name.String() == schema {
```

Comment:
Do we need to make this check case-sensitive? 🤔️

Reply:
The current impl is case-sensitive 😬